llvm/tools/llvm-profgen/PerfReader.cpp

```diff
@@ -1062,6 +1132,18 @@ bool PerfScriptReader::isLBRSample(StringRef Line) {
   Line.trim().split(Records, " ", 2, false);
   if (Records.size() < 2)
     return false;
+  // Check if there is an event name before the leading IP.
```

Comment:
Update the header comment for this function to include a representation of LBR sample with event name